packages/react-charting/src/utilities/utilities.ts

```diff
@@ -387,6 +387,44 @@ export function createYAxisForOtherCharts(yAxisParams: IYAxisParams, isRtl: bool
     .tickPadding(tickPadding)
     .tickValues(domainValues)
     .tickSizeInner(-(containerWidth - margins.left! - margins.right!));
+
+  yAxisTickFormat ? yAxis.tickFormat(yAxisTickFormat) : yAxis.tickFormat(d3Format('.2~s'));
+  yAxisElement ? d3Select(yAxisElement).call(yAxis).selectAll('text').attr('aria-hidden', 'true') : '';
+  axisData.yAxisDomainValues = domainValues;
+  return yAxisScale;
```

Comment:
instead of creating a net new function we can add a parameter and use the existing function only.